src/index.html

```diff
@@ -155,10 +155,9 @@
         <div class="mainDrawer-scrollContainer scrollContainer focuscontainer-y"></div>
     </div>
     <div class="skinHeader focuscontainer-x"></div>
-    <div class="mainAnimatedPages skinBody">
+    <div id="reactRoot" class="mainAnimatedPages skinBody">
```

Comment:
Was this required? I was concerned using the same element for both routers could cause some weird edge cases to arise.

Reply:
the reason I added here b/c of skinBody style.

Reply:
update: 732e3eb